lfucg-exactions/server/plats/serializers.py

```diff
@@ -13,6 +13,12 @@ def get_cleaned_gross_acreage(self, obj):
         set_acreage = str(obj.gross_acreage).rstrip('0').rstrip('.')
         return set_acreage
 
+    # EXAMPLE FOR FIELD VALIDATION
```

Comment:
still needed?

Reply:
We are potentially using this method to validate entries in payments and ledgers so I wanted to leave an example in for future me. I can remove it if we need to, just have another ticket open where it could be useful.

Reply:
@Wei-KeatKong - how do you feel about that comment?